frontend/src/config/constants.ts

```diff
@@ -13,17 +13,18 @@ import { getMostRecentPastTerm } from "pages/TermPlanner/utils";
  */
 export const REDUX_PERSIST_VERSION = 3;
 
-export const FEEDBACK_LINK = "https://docs.google.com/forms/d/e/1FAIpQLSdGoxb-1fRRLySEi5j4Gy_ERWt3XWTBwyKJaDWlTqhy8Xdlxw/viewform?usp=sf_link";
+export const FEEDBACK_LINK = 'https://docs.google.com/forms/d/e/1FAIpQLSdGoxb-1fRRLySEi5j4Gy_ERWt3XWTBwyKJaDWlTqhy8Xdlxw/viewform?usp=sf_link';
 
-export const TERM = `${new Date().getFullYear() + (getMostRecentPastTerm().T === 3 ? 1 : 0)}-T${(getMostRecentPastTerm().T + 1) % 3}`;
+export const CURR_YEAR = new Date().getFullYear();
+export const TERM = `${CURR_YEAR + (getMostRecentPastTerm(CURR_YEAR).T === 3 ? 1 : 0)}-T${(getMostRecentPastTerm(CURR_YEAR).T + 1) % 3}`;
 export const TIMETABLE_API_URL = `https://timetable.csesoc.app/api/terms/${TERM}/courses`;
 
 // Global colors - currently only being used in LiquidProgressChart
-export const lightYellow = "#f9b01e";
-export const lightGrey = "#565652";
-export const darkGrey = "#323739";
-export const yellow = "#FAAD14";
-export const purple = "#9254de";
-export const lightPurple = "#efdbff";
+export const lightYellow = '#f9b01e';
+export const lightGrey = '#565652';
+export const darkGrey = '#323739';
+export const yellow = '#FAAD14';
+export const purple = '#9254de';
+export const lightPurple = '#efdbff';
 
-export const inDev = import.meta.env.VITE_ENV === "dev";
+export const inDev = import.meta.env.DEV;
```

Comment:
isnt there also `NODE_ENV` or something?

Reply:
used `import.meta.env.MODE` for env comparison instead